tests/test_core.py

```diff
@@ -36,64 +38,111 @@ def test_inspect_dummy_execs():
 
 
 @pytest.mark.parametrize(
-    'package_path, expected_path',
+    'package_path, expected_path, build_args_envs, dry_run',
     [
         (
             Path(__file__).parent / 'cases' / 'executor_1',
             Path(__file__).parent / 'cases' / 'executor_1.json',
+            { 
+                'AUTH_TOKEN': "AUTH_TOKEN",
+                'TOKEN': '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'
```

Comment:
is it real Github token?

Reply:
yes it is the token of my other account, i will replace it